### ISSUE TITLE
Insertion point not visible upon creation of new document

### ISSUE THREAD

**Issue by Geda173** [outline/outline#5643]
When first creating a new document the insertion point in the body is not visible. New text is not immediately visible. The page has to be refreshed for everything to function as expected.

**To Reproduce**
1. Go to "new document" OR visit an already created document that has no text in it.
2. Click on the section where the body is supposed to be.
3. Attempt to write text. The insertion point will not be visible. Text that you have produced will also not be visible.
4.  Refresh the page. Everything will be visible and working as expected

**Expected behavior**
The insertion point in the body and text should immediately be visible. 

**Screenshots**
New document:
![image](https://github.com/outline/outline/assets/23613827/299a3719-ca71-47b3-87bc-08cfd5ac93f7)
After refreshing the page:
![image](https://github.com/outline/outline/assets/23613827/b1c4f7f0-71cc-4691-8ea1-1851df956c0d)

**Outline (please complete the following information):**
- Install: selfhosted
- Version: v0.72.0

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Brave (+ other chromium based browsers)
 - Version 1.56.14

**Comment by tommoor** [comment]
You can see in the first screenshot that the client was unable to connect to the collaboration server, please post the console logs from the browser.

**Comment by Geda173** [comment]
It was a bunch of websocket errors: 

`hocuspocus-provider.esm.js:1918 WebSocket connection to 'wss://subdomain.mydomain.com/collaboration/document.7cbf79cd-09bb-49ca-8fa9-c98448ecd78d' failed:"`

Turns out nginx was not properly configured. I needed to "enable" websockets in nginx as per [a suggestion in #5620
](https://github.com/outline/outline/issues/5620#issuecomment-1662888085).

Thank you very much for the prompt response.
